### DISCUSSION TITLE
How to create list of strings in schema Column?

### DISCUSSION THREAD

**Discussion by itsezlife** [powersync-ja/powersync.dart#18] [Q&A] [answered]
I need to store list of strings in table in schema, however there are only text, real and integer options. Is there a support from PowerSync to do so?

**Comment by rkistner** [comment, marked as answer]
You can use `json`, `jsonb`, `text[]`, or any other array type. PowerSync converts all of those to JSON text, which you can then query using SQLite's [json1](https://www.sqlite.org/json1.html) functions, or parse in the app code.

**Reply by itsezlife** [reply]
Thanks for the response! Will take it into account. However, I decided to encode list of strings in order to store as string and then decode it on the client side back into string.
